esrally/mechanic/telemetry.py

```diff
@@ -190,7 +186,7 @@ def instrument_env(self, car, candidate_id):
         java_opts = self.java_opts(log_file)
 
         self.logger.info("jfr: Adding JVM arguments: [%s].", java_opts)
-        return {"ES_JAVA_OPTS": java_opts}
+        return java_opts.split()
```

Comment:
I think we just need to return `java_opts` here? A Python list does not provide a `split` method?

Reply:
@danielmitterdorfer  For jfr version of this method, I kept java_opts as string and then split it into list by white space, the code seemed simpler that way. But I can change it to use list from the beginning. What do you think?

Reply:
actually nevermind, it seems to work fine if i keep it as a string and return java_opts..

Reply:
java_opts = self.java_opts(log_file) for FlightRecorder still returns string, and here I split it into list. Do you think I should change java_opts to return list instead to make it more clear?

Reply:
Yes, I think we should always return lists.